reference/docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -126,3 +126,9 @@ create unique index dash_lock_ts_idx on dash_task_lock (task, server_id);
 INSERT INTO SAKAI_SITE_PAGE VALUES('!user-99', '!user', 'Dashboard', '0', 0, '0' );
 INSERT INTO SAKAI_SITE_TOOL VALUES('!user-999', '!user-99', '!user', 'sakai.dashboard', 1, 'Dashboard', NULL );
 
+
+--
+-- SAK-31636 Rename existing 'Home' tools
+--
+
+update sakai_site_page set title = 'Overview' where title = 'Home';
```

Comment:
Should be `update SAKAI_SITE_PAGE set title = 'Overview' where title = 'Home';` as us silly MySQL people have case sensitive databases.